pkg/providers/instancetype/suite_test.go

```diff
@@ -466,6 +466,7 @@ var _ = Describe("InstanceType Provider", func() {
 			nodeClass.Spec.Kubelet = &v1alpha2.KubeletConfiguration{
 				ImageGCHighThresholdPercent: lo.ToPtr(int32(30)),
 				ImageGCLowThresholdPercent:  lo.ToPtr(int32(20)),
```

Comment:
Do we also want to add a test for the default/expected setting?

Reply:
Maybe - as part of general KubeletConfiguration tests